docs/zh-cn/dev/principals/dummy.md

```diff
@@ -105,9 +105,9 @@ public final class Version {
             if (version == null || version.length() == 0) {  
                 // 如果MANIFEST.MF规范中没有版本号，基于jar包名获取版本号  
                 String file = cls.getProtectionDomain().getCodeSource().getLocation().getFile();  
-                if (file != null && file.length() > 0 && file.endsWith(".jar")) {  
+                if (file != null &amp;&amp; file.length() &gt; 0 &amp;&amp; file.endsWith(".jar")) {  
```

Comment:
You shouldn't change here. What you should change is Line 37 and Line 43

Reply:
OK，I have already corrected.